src/providers/Onboarding.js

```diff
@@ -10,11 +10,12 @@ const DEFAULT_CONFIG = {
     name: '',
     description: '',
     logo: null,
-    logotype: null,
+    logo_type: null,
+    token_logo: null,
```

Comment:
Are you implying we use  `token_logo` as well in the `dao-list` repo ?

Note that we are currently using `token: { logo }`

Reply:
yes the intention is to just have it on the root since we don't have any more attr